models/user.js

```diff
@@ -1,15 +1,17 @@
 'use strict';
 module.exports = function(sequelize, DataTypes) {
-  var User = sequelize.define('User', {
+  return sequelize.define('User', {
     email: DataTypes.STRING,
     email_verified: DataTypes.BOOLEAN,
     password: DataTypes.STRING
   }, {
     classMethods: {
       associate: function(models) {
+        console.log(models)
         // associations can be defined here
+        models.User.hasMany( models.UserTopic )
+        models.User.belongsToMany( models.Topic, { through: 'UserTopics' } )
```

Comment:
I don't think we need the relationship to exist in this direction - we're only concerned with the UserTopics a given user has; the reverse relationship does not make sense.